### PR TITLE
Czml properties changed as fillcolor

### PR DESCRIPTION
**Pull request by esraerik**
#5705 changed the example to reference to the fill color instead of outlineColor

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@esraerik thanks for the pull request!

I noticed that [`CHANGES.md`](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the Cesium API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and bump this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

__I am a bot who helps you make Cesium awesome!__ Thanks again.

File: Apps/Sandcastle/gallery/CZML Reference Properties.html [hunk 1]
```diff
@@ -28,82 +28,80 @@
...
-    "name" : "Referencing Position",
-    "description" : "<p>For more examples of reference properties, see CZML Polygon - Interpolating References.</p>",
-    "billboard" : {
-        "image" : "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAACvSURBVDhPrZDRDcMgDAU9GqN0lIzijw6SUbJJygUeNQgSqepJTyHG91LVVpwDdfxM3T9TSl1EXZvDwii471fivK73cBFFQNTT/d2KoGpfGOpSIkhUpgUMxq9DFEsWv4IXhlyCnhBFnZcFEEuYqbiUlNwWgMTdrZ3JbQFoEVG53rd8ztG9aPJMnBUQf/VFraBJeWnLS0RfjbKyLJA8FkT5seDYS1Qwyv8t0B/5C2ZmH2/eTGNNBgMmAAAAAElFTkSuQmCC",
-        "scale" : 1.5
-    },
-    "label" : {
-        "fillColor" : {
-            "rgba" : [255, 255, 255, 255]
+    var czml = [{
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/CZML Reference Properties.html]: This shows up in Sandcastle as everything having an extra tab on the left.

File: Apps/Sandcastle/gallery/CZML Reference Properties.html [hunk 2]
```diff
@@ -28,82 +28,80 @@
...
+                "cartographicDegrees" : [
+                    -115.0, 37.0, 0,
+                    -115.0, 32.0, 0,
+                    -107.0, 33.0, 0,
+                    -102.0, 31.0, 0,
+                    -102.0, 35.0, 0
+                ]
+            },
+            "height" : 0,
+            "fillColor" : {
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/CZML Reference Properties.html]: The fill color is not specified by `"fillColor"`, it'd be more like this:

```
"material" : {
            "solidColor" : {
                "color" : {
                    "reference" : "fillColor-reference#label.fillColor"
                }
            }
        }
```

See the CZML docs on polygons: https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Polygon

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 Change Log
 ==========
+### 1.36 - 2017-08-17
```
Comment (ggetz) [inline CHANGES.md]: This would be `### 1.37 - 2017-09-01`, and it's not a breaking change.

**Comment by hpinkos** [comment]
@ggetz can you review this?  I updated the example

File: Apps/Sandcastle/gallery/CZML Reference Properties.html [hunk 4]
```diff
@@ -10,10 +10,12 @@
...
-    require.config({
-        baseUrl : '../../../Source',
-        waitSeconds : 60
-    });
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/CZML Reference Properties.html]: Should this be here? It's not in any of the other sandcastle examples.

**Comment by ggetz** [comment]
@hpinkos Just that one comment

File: Apps/Sandcastle/gallery/CZML Reference Properties.html (hunk 4 above)
Comment (hpinkos) [reply Apps/Sandcastle/gallery/CZML Reference Properties.html]: @ggetz that was generated when I saved the files directly from Sandcastle so I think it's intentional

**Pull request merged**